src/Avalonia.Controls/StackPanel.cs

```diff
@@ -52,6 +58,15 @@ public Orientation Orientation
             set { SetValue(OrientationProperty, value); }
         }
 
+        /// <summary>
+        /// Gets or sets if the child controls will be layed out in reverse order.
+        /// </summary>
+        public bool ReverseOrder
```

Comment:
This property seems rather specific to this use-case to be adding a property to `StackPanel`. I think if the property also affected the layout of the controls (similar to how [FlowDirection](https://wpf.2000things.com/2014/04/02/1042-how-flowdirection-affects-a-stackpanel/) works with WPF, but for vertical layouts too) I might be able to get on board with it, but just reversing the order of the items feels a bit niche.

Reply:
I think there should be some ItemsControl that transforms notifications into their visual representation. This would also make it possible to just bind a list of notifications to it and lets you just insert notifications at the top position and therefore makes this property redundant.

Reply:
As discussed using a ReversibleStackPanel control until we support FlowDirection.